documentation/onboard-service.md

```diff
@@ -1,5 +1,6 @@
 # Onboard a new service to AKS
 
+[link to form](onboard-form-template.md)
```

Comment:
Before starting, it is important to capture the information required upfront using the [Onboarding form](onboard-form-template.md) even if it is subject to change in the future.